### PR TITLE
Quad 150 brand

### PR DESCRIPTION
**Pull request by afromme**
This PR includes a new page for brand use guidelines, and updates to the previous guidelines. (The previous brand use guidelines were included as a PDF)

### CODE REVIEW

File: about/brand_use.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+### Further guidelines based on project type:
+
+#### If you are an author with published works on arXiv
+
+For arXiv authors only, non-commercial, personal, and minimal use of the arXiv logo for the purposes of linking from your website to arXiv’s website is allowed. In these instances the logo should follow these guidelines:
+1. The logo is displayed at a size between a minimum of 50px and a maximum of 100px in width.
+2. The logo is used with the intention of linking to arXiv (for example to your papers), and not for branding purposes.
+3. The logo must link to arXiv.
+
+Download the appropriate file for your personal use as an author: [png](https://cornell.box.com/v/arxiv-logo-small-png) or [jpg](https://cornell.box.com/v/arxiv-logo-small-jpg)
```
Comment (jweiskoff) [inline about/brand_use.md:30]: Should we host these files on arXiv rather than in a Cornell box?

**Review by jweiskoff — APPROVED** [review]
Good to go as is. my question is more of a potential consideration for the future, and not something that should be considered a blocker for now.

File: about/brand_use.md (hunk 1 above)
Comment (afromme) [reply about/brand_use.md:30]: Yes, in the future, that is the plan!

**Review by bmaltzan — APPROVED** [review]

File: about/brand_use.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+
+#### For products that use the arXiv API
+
+1. Acknowledge arXiv data usage with this statement on your product: “Thank you to arXiv for use of its open access interoperability. This [service/ product] was not reviewed or approved by, nor does it necessarily express or reflect the policies or opinions of, arXiv.”
+2. If your product is linking directly to arXiv PDFs or abstract pages, you may either use the full arXiv logo or the logomark. (Note: your product must also comply with [arXiv’s license agreements](/help/license))
```
Comment (SBBCornell) [inline about/brand_use.md:36]: should we link to the logo mark use guidelines?

**Review by SBBCornell — APPROVED** [review]
I just left two small comments. Looks good!

File: about/brand_use.md (hunk 2 above)
Comment (afromme) [reply about/brand_use.md]: Good call. We will do that when we move to phoenix. For now, I added a note to email us for details.

**Pull request merged**